product_docs/docs/pgd/5.7/upgrades/manual_overview.mdx

```diff
@@ -59,9 +59,18 @@ nodes are at the newer release. There must be no more than two versions of the
 software running at the same time. An upgrade must be completed, with all nodes
 fully upgraded, before starting another upgrade.
 
-An upgrade process can take more time when 
-caution is required to reduce business risk. However, we don't recommend
-running mixed versions of the software indefinitely.
+Where additional caution is required to reduce business risk, more time may be
+required to perform an upgrade. For maximum caution and to reduce the time
+required upgrading production systems, we suggest performing the upgrades in a
+separate test environment first. 
+
+Don't run with mixed versions of the software for any longer than is absolutely
+necessary to complete the upgrade. You can check on the versions in the cluster
+using the [`pgd show-versions`](/pgd/latest/cli/command_ref/pgd_show-version/)
```

Comment:
Is it pgd show-version**s** or pgd show-version? The URI suggests single version (not a plural).